src/pyskindose/table_data/plot_HVL_data.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from pathlib import Path
+import os
+import plotly.graph_objects as go
+
+
+data_path = Path(os.path.abspath(__file__)).parent
+file_name = "HVL_simulated.csv"
+
+data = pd.read_csv(data_path / file_name)
+
+data_axiom_single = []
+data_allura_single = []
+
+filtration_axiom = [0, 0.1, 0.2, 0.3, 0.6, 0.9]
+
+
+filtration_allura_al = [0, 1, 1, 1]
+filtration_allura_cu = [0, 0.1, 0.4, 0.9]
+
+for i in range(len(filtration_allura_al)):
```

Comment:
If this file is to be save I think these for-loops should be change into list comprehensions

Reply:
plot script removed, we dont need it